drivers/SmartThings/zwave-switch/src/zooz-zen-30-dimmer-relay/init.lua

```diff
@@ -86,6 +83,71 @@ local function can_handle_zooz_zen_30_dimmer_relay_double_switch(opts, driver, d
   return false
 end
 
+local function find_child(parent, src_channel)
+  if src_channel == 0 then
+    return parent
+  else
+    return parent:get_child_by_parent_assigned_key(string.format("%02X", src_channel))
+  end
+end
+
+local function component_to_endpoint(device, component)
+  return { ENDPOINTS.dimmer }
+end
```

Comment:
I think this results in everything being sent to component 0, right?

Reply:
everything from parent. Child devices doesn't use this mapping

Reply:
It needs to be here because if I will not return {0} here then all commands will be sent to dst_channels = {}

Reply:
I guess it gets used in the level defaults and the child doesn't support that so it doesn't matter there. I think I get it now.